src/components/Tabs/style.scss

```diff
@@ -2,7 +2,7 @@
 
 .tabs {
   padding: 14px 0 0 14px; /* so first tab aligns with labels */
-  border-bottom: 1px solid $cGray200;;
+  border-bottom: 1px solid $cGray200;
```

Comment:
Can we use the new color convention here? I should mark more clearly the deprecated colors, but we are moving to the systematized colors. This would probably map to $cGray40.

Reply:
def, good to know